packages/mds-metrics/request-handlers.ts

```diff
@@ -198,8 +198,15 @@ export async function getEventCounts(req: MetricsApiRequest, res: GetEventCounts
   and finer-grained field-fetching a la GraphQL.
 */
 export async function getAll(req: MetricsApiRequest, res: GetAllResponse) {
-  const { body } = req
+  const { body, query } = req
   const slices = getTimeBins(body)
+  const geography_id = query.geography_id || null
+  const provider_id = query.provider_id || null
+
+  if (geography_id !== null && !isUUID(geography_id))
+    return res.status(400).send(new ServerError(`geography_id ${geography_id} is not a UUID`))
```

Comment:
ServerError isn't the proper error type to use here, should be a `BadParamsError`

Reply:
`BadParamsError` and `ValidationError` are largely similar and should probably be consolidated.